static/modules/gamemanager.js

```diff
@@ -123,6 +130,16 @@ class GameManager {
       }
     });
 
+    document.addEventListener("keydown", e => {
```

Comment:
You added a pause button to `buttons.js` (which is the proper way to do this), but then hard-coded it to being "p" anyway? Use the existing buttons system so the pause key can be rebound and bound to a controller input

Reply:
Actually just leave it, I was planning on making the fullscreen function a proper button so I'll fix this at the same time

Reply:
We don't do the "proper" way for fullscreen either and I wasn't sure how to implement it, so I did it the easy way and left the button.js code in for whoever fixes it.